### PR TITLE
Add references to hackathon march 2022 presentations 

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

File: markdown/events/2022/hackathon-march-2022.md [hunk 1]
```diff
@@ -99,6 +101,15 @@ This group will focus on restructuring and organising the documentation on the w
 - Restructure website navigation and improve the user experience.
 - Act in a support role for technical help to others in the hackathon.
 
+## Talks
+
+We will also be offering three different talks throughout the event at 14:30 each day.
+These focus on useful 'soft skills' for developers that can enhance their documentation.
+
+🖉 "Draw stuff good: how to use Inkscape" by [Zandra Fagernäs](https://twitter.com/ZandraSelina)<br>
```
Comment (mashehu) [inline markdown/events/2022/hackathon-march-2022.md:109]: this emoji doesn't seem to get rendered in github nor gitpod.

Comment (jfy133) [inline markdown/events/2022/hackathon-march-2022.md:109]:
```suggestion
🖊 "Draw stuff good: how to use Inkscape" by [Zandra Fagernäs](https://twitter.com/ZandraSelina)<br>
```

Comment (jfy133) [reply markdown/events/2022/hackathon-march-2022.md]: @mashehu updated, can youcheck again?

**Comment by jfy133** [comment]
Markdown links are for cutandrun and mhcquant :thinking:

File: markdown/events/2022/hackathon-march-2022.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ testing each command making sure they work, checking phrasing and descriptions t
 To keep things manageable, we will organise into the groups.
 You are free to change groups as you wish during the hackathon.
 
+For the first time, we will also have talks and presentations during the hackathon.
```
Comment (ewels) [inline markdown/events/2022/hackathon-march-2022.md:48]: Not sure it's really the first time?

Comment (jfy133) [reply markdown/events/2022/hackathon-march-2022.md:48]: Ah ok... I didn't remember any non-n-core ones. Sorry! Will remove

File: markdown/events/2022/hackathon-march-2022.md [hunk 3]
```diff
@@ -45,6 +45,8 @@ testing each command making sure they work, checking phrasing and descriptions t
...
+These will be focused on making pipeline-related graphics and customising MultiQC reports. Please see [below](#talks) for more information.
 ## Group 1 - Pipeline documentation
```
Comment (jfy133) [inline markdown/events/2022/hackathon-march-2022.md:50]:
```suggestion
We will also have talks and presentations during the hackathon.
These will be focused on making pipeline-related graphics and customising MultiQC reports. Please see [below](#talks) for more information.

## Group 1 - Pipeline documentation
```

**Review by mashehu — APPROVED** [review]

File: markdown/events/2022/hackathon-march-2022.md [hunk 4]
```diff
@@ -99,6 +102,15 @@ This group will focus on restructuring and organising the documentation on the w
...
 - Act in a support role for technical help to others in the hackathon.
 
+## Talks
+
+We will also be offering three different talks throughout the event at 14:30 each day.
+These focus on useful 'soft skills' for developers that can enhance their documentation.
+
+🖊 "Draw stuff good: how to use Inkscape" by [Zandra Fagernäs](https://twitter.com/ZandraSelina)<br>
+🚇 "Creating pipeline diagrams in Inkscape (like that [fancy metro-map](https://raw.githubusercontent.com/nf-core/eager/master/docs/images/usage/eager2_metromap_complex.png) one)" by [James Fellows Yates](https://github.com/jfy133)<br>
+📊 "Pimp your MultiQC report" by [Phil Ewels](https://github.com/ewels)<br>
```
Comment (ewels) [inline markdown/events/2022/hackathon-march-2022.md:112]:
```suggestion
* 🖊 "Draw stuff good: how to use Inkscape" by [Zandra Fagernäs](https://twitter.com/ZandraSelina)
* 🚇 "Creating pipeline diagrams in Inkscape (like that [fancy metro-map](https://raw.githubusercontent.com/nf-core/eager/master/docs/images/usage/eager2_metromap_complex.png) one)" by [James Fellows Yates](https://github.com/jfy133)
* 📊 "Pimp your MultiQC report" by [Phil Ewels](https://github.com/ewels)
```

**Review by ewels — APPROVED** [review]

**Pull request merged**